Source/GSXibKeyedUnarchiver.m

```diff
@@ -1033,4 +1097,26 @@ - (NSDictionary *) decoded
 {
   return decoded;
 }
+
+- (NSDictionary *) customClasses
+{
+  return _customClasses;
+}
+
+- (void) createCustomClassRecordForId: (NSString *)theId
+                      withParentClass: (NSString *)parentClassName
+                       forCustomClass: (NSString *)customClassName
+{
+  if (theId == nil || customClassName == nil)
+    return;
+
+  NSMutableDictionary *dict =
+    [NSMutableDictionary dictionaryWithObjectsAndKeys:
+                           parentClassName, @"parentClassName",
+                         theId, @"id",nil];
```

Comment:
if you need the class name separately in Gorm you need to add it here.

Reply:
Nope, it is not needed separately.